gluonnlp/model/language_model.py

```diff
@@ -233,6 +234,64 @@ def forward(self, inputs, begin_state=None): # pylint: disable=arguments-differ
         out = self.decoder(encoded)
         return out, state
 
+def detach(hidden):
+    if isinstance(hidden, (tuple, list)):
+        hidden = [detach(h) for h in hidden]
+    else:
+        hidden = hidden.detach()
+    return hidden
+
+def forward(model, inputs, begin_state=None):
```

Comment:
leave this in script

Reply:
I think detach hidden is a basic utility in language model, since it is used in basic word level langauge model, awd, cache, elmo.